feature/center/src/main/java/com/mifos/feature/center/center_list/ui/CenterListScreen.kt

```diff
@@ -149,8 +153,9 @@ fun CenterListScreen(
                     actions = {
                         FilledTonalButton(
                             onClick = {
+                                sync.value = true
                                 syncClicked(selectedItems.toList())
```

Comment:
we are not using it anymore. Remove it. 
line no 114 as well

Reply:
right, ill make those necessary changes